### DISCUSSION TITLE
Set default cite style

### DISCUSSION THREAD

**Discussion by AdiWeit** [typst/typst#2577] [General]
### Description

It would be great to be able to set a standard citation style, so you do not have to add the style parameter to every cite you want to do with the in your document commonly used style. The style could be overwritten if the parameter is entered in the #cite(...) method. 
The cetz library uses the "set-style" method in the same way, but for graphs. 
Maybe I am missing something and this is already an option, but I did not find it, so in that case it should be added to the documentation in my opinion. 

### Use Case

Like I said it would be helpful if you have many citations you want to do in the same style without having to specify the style every time.

**Comment by Enivex** [comment]
`#set cite(style: "style goes here")`
